graylog2-web-interface/src/views/components/aggregationbuilder/EventListConfiguration.jsx

```diff
@@ -0,0 +1,34 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { Checkbox, FormGroup } from 'components/graylog';
+
+type Props = {
+  enabled: boolean,
+  onChange: (value: boolean) => void,
+};
+
+const EventListConfiguration = ({ enabled, onChange }: Props) => {
+  return (
+    <form>
+      <FormGroup>
+        {/* eslint-disable-next-line no-undef */ /* $FlowFixMe: checked is part of target */}
+        <Checkbox onChange={(event: SyntheticEvent<HTMLInputElement>) => onChange(event.target.checked)} checked={enabled}>
```

Comment:
Is there a reason, you are using `SyntheticEvent` here? With `SyntheticInputEvent`, we don't need the $FlowFixMe comment

Reply:
I did not know :D